Packages/com.unity.inputsystem/InputSystem/Plugins/XR/XRSupport.cs

```diff
@@ -317,6 +318,7 @@ public static void Initialize()
 
             InputSystem.onFindLayoutForDevice += XRLayoutBuilder.OnFindLayoutForDevice;
 
+#if ENABLE_VR
 #if !DISABLE_BUILTIN_INPUT_SYSTEM_WINDOWSMR
```

Comment:
These should all be safe to delete by now.  They pass the two year mark since all XR.SDK packages include their own layouts.

Reply:
removed with layouts.